Cython/Utility/ModuleSetupCode.c

```diff
@@ -1796,3 +1796,18 @@ static void __Pyx_FastGilFuncInit(void) {
 }
 
 #endif
+
+///////////////////// UtilityCodePragmas /////////////////////////
+
+#if _MSC_VER
+#pragma warning( push )
+#pragma warning( disable : 4127 )
```

Comment:
If we include the C comment in the closing part in the output (which IMHO seems ok to do), should we include a comment here, too, about the kind of warning that we suppress?

Reply:
I've changed the comment here so that it does appear. The C comment in the closing part doesn't appear in the output though.

Reply:
Ah, right. We remove all "simple" comments.